mlrun/run.py

```diff
@@ -453,8 +453,8 @@ def import_function(url="", secrets=None, db="", project=None, new_name=None):
 
     special URLs::
 
-        function hub: hub://{name}[:{tag}]
-        local mlrun db:       db://{project-name}/{name}[:{tag}]
+        function hub:       hub://[{source}/]{name}[:{tag}]
```

Comment:
Just to make sure - the function name cannot contain `/` and it's verified elsewhere, right? Otherwise this is not deterministic.

Reply:
added a try except block for that